include/fmt/chrono.h

```diff
@@ -1337,9 +1456,28 @@ template <typename OutputIt, typename Char> class tm_writer {
     if (is_classic_ || ns == numeric_system::standard) return write2(tm_min());
     format_localized('M', 'O');
   }
+
   void on_second(numeric_system ns) {
-    if (is_classic_ || ns == numeric_system::standard) return write2(tm_sec());
-    format_localized('S', 'O');
+    if (is_classic_ || ns == numeric_system::standard) {
+      write2(tm_sec());
+      if (has_subsecs_) {
+        if (std::is_floating_point<Rep>::value) {
+          auto buf = memory_buffer();
+          format_sec_fractional<Rep, Period::num, Period::den>(
+              buf, subsecs_.count());
```

Comment:
subsecs should be passed as duration and template arguments deduced.

Reply:
done